packages/framework/dailyTableUpdates/table_Automation.py

```diff
@@ -0,0 +1,81 @@
+#/usr/bin/env python3
+"""
+      Trilinos Status Table Automation Script
+      
+      tablestatus.py <pr_status, pr_merged, failed_pr, 
+                                waiting_pr, open_pr, mm_status, successful_mm, jira_ticket>
+"""
+from datetime import timedelta
+from datetime import date
+from time import sleep as pause
+import sys
+import os
+
+STOP = 0b11
+try:
+  import pytablewriter as ptw
+except ImportError:
+  print("Trying to Install required module(s)\n\n\n")
+  pause(STOP)
+  os.system('cls' if os.name == 'nt' else 'clear')
+  os.system('python3 -m ensurepip')
+  os.system('python3 -m pip install pytablewriter')
```

Comment:
Please remove this try-except block and instead document the dependencies and how to install them at the top of this script.